composer/datasets/imagenet.py

```diff
@@ -100,12 +101,12 @@ def initialize_object(self, batch_size: int, dataloader_hparams: DataLoaderHpara
                         raise ValueError(
                             "datadir is required if use_synthetic is False and ffcv_write_dataset is True.")
                     ds = ImageFolder(os.path.join(self.datadir, split))
-                    create_ffcv_dataset(dataset=ds,
-                                        write_path=dataset_filepath,
-                                        max_resolution=500,
-                                        num_workers=dataloader_hparams.num_workers,
-                                        compress_probability=0.50,
-                                        jpeg_quality=90)
+                    write_ffcv_dataset(dataset=ds,
+                                       write_path=dataset_filepath,
+                                       max_resolution=500,
+                                       num_workers=dataloader_hparams.num_workers,
+                                       compress_probability=0.50,
```

Comment:
I am wondering if "probability" might be canonically shortened to "prob"

Reply:
To match with what ffcv has let us keep the full name. https://github.com/libffcv/ffcv/blob/main/ffcv/fields/rgb_image.py#L294